src/audio/component.c

```diff
@@ -234,3 +234,27 @@ void audio_stream_copy_to_linear(const struct audio_stream __sparse_cache *sourc
 		snk += bytes_copied;
 	}
```

Comment:
In commit text "issue is WAed", what does it mean, is it a typo?

Reply:
I think it is WAed "worked around", but better to avoid acronyms in git messages. @btian1 maybe instead of "Due to urgent schedule" put "Previously performance logs were disabled in Zephyr builds due to this xcc limitation. Fix the issue by moving comp_copy() implementation out of the header. Also du eto all cmocka..."

Reply:
thanks, @aiChaoSONG @kv2019i @lgirdwood , combined all your input and then modified commit message again, I think this time it should looks better, :)